pkg/schema/spec/default-definitions.yaml

```diff
@@ -23,9 +23,15 @@
 - vspecName: Vehicle.CurrentLocation.Longitude
   requiredPrivileges:
     - VEHICLE_ALL_TIME_LOCATION
+- vspecName: Vehicle.CurrentLocation.Heading
+  requiredPrivileges:
+    - VEHICLE_ALL_TIME_LOCATION
 - vspecName: Vehicle.Exterior.AirTemperature
   requiredPrivileges:
     - VEHICLE_NON_LOCATION_DATA
+- vspecName: Vehicle.IsIgnitionOn
+  requiredPrivileges:
+    - VEHICLE_NON_LOCATION_DATA
```

Comment:
Is this location data? I don't think it is.

Reply:
it is not location data

Reply:
@elffjs ?

Reply:
I think I misread.